src/Core/Content/Media/Thumbnail/ThumbnailService.php

```diff
@@ -158,7 +166,18 @@ private function createThumbnailsForSizes(
             return 0;
         }
 
-        $mediaImage = $this->getImageResource($media);
+        try {
```

Comment:
Could you move this into the Command and add a flag to ignore the Exceptions? The Service should still throw the exception.